app/views/stories/_form.html.erb

```diff
@@ -73,54 +73,50 @@ Please don't use this to promote the story, summarize the post, or explain why y
 See the guidelines below for more." %>
     </div>
 
-    <div class="boxline actions markdown_help_toggler">
-      <a href="#" id="story_guidelines_toggler">
-        Story submission guidelines
-      </a>
-      <div id="story_guidelines" style="<%= show_guidelines?? "" : "display: none;" %>">
```

Comment:
We could keep this `<div>`, and prevent the re-indentation of the guidelines. This would make `git blame` cleaner. Not sure it's worth the trouble, but I'm open to doing that.